week-2/implement/tally.test.js

```diff
@@ -29,3 +29,26 @@
 // Given an invalid input like a string
 // When passed to tally
 // Then it should throw an error
+const countString = require("./tally.js");
+
+test('test if the object counts',function()
+{
+const arr = ["a","b","c","a","d","e","b","c","c","c"];
+const currentOutput= {"a": 2, "b":2, "c": 4, "d":1, "e":1 };
+const targetOutput= countString(arr);
+expect(currentOutput).toEqual(targetOutput);
+})
+test('test if the object counts',function()
```

Comment:
This test name could be a bit more specific, so it tells us that it tests for an _empty_ array as an input